packages/database/src/core/PersistentConnection.ts

```diff
@@ -935,8 +935,8 @@ export class PersistentConnection extends ServerActions {
     if (this.securityDebugCallback_) {
       this.securityDebugCallback_(body);
     } else {
-      if ('msg' in body && typeof console !== 'undefined') {
-        console.log('FIREBASE: ' + body['msg'].replace('\n', '\nFIREBASE: '));
+      if ('msg' in body) {
+        log('FIREBASE: ' + body['msg'].replace('\n', '\nFIREBASE: '));
```

Comment:
I believe this was explicitly meant to be logged without turning on logging.  Essentially you can use a special auth token and the backend will send back debug messages that get automatically logged to the console.

Reply:
SGTM